shortcut_composer/api_krita/enums/action.py

```diff
@@ -0,0 +1,581 @@
+    H_MIRROR_ACTION = "hmirror_action" # Horizontal Mirror Tool
+    V_MIRROR_ACTION = "vmirror_action" # Vertical Mirror Tool
+    NEXT_BLENDING_MODE = "Next Blending Mode" # Next Blending Mode
+    PREVIOUS_BLENDING_MODE = "Previous Blending Mode" # Previous Blending Mode
+    SELECT_NORMAL_BLENDING_MODE = "Select Normal Blending Mode" # Select Normal Blending Mode
+    SELECT_DISSOLVE_BLENDING_MODE = "Select Dissolve Blending Mode" # Select Dissolve Blending Mode
+    SELECT_BEHIND_BLENDING_MODE = "Select Behind Blending Mode" # Select Behind Blending Mode
+    SELECT_CLEAR_BLENDING_MODE = "Select Clear Blending Mode" # Select Clear Blending Mode
+    SELECT_DARKEN_BLENDING_MODE = "Select Darken Blending Mode" # Select Darken Blending Mode
+    SELECT_MULTIPLY_BLENDING_MODE = "Select Multiply Blending Mode" # Select Multiply Blending Mode
+    SELECT_COLOR_BURN_BLENDING_MODE = "Select Color Burn Blending Mode" # Select Colour Burn Blending Mode
+    SELECT_LINEAR_BURN_BLENDING_MODE = "Select Linear Burn Blending Mode" # Select Linear Burn Blending Mode
+    SELECT_LIGHTEN_BLENDING_MODE = "Select Lighten Blending Mode" # Select Lighten Blending Mode
+    SELECT_SCREEN_BLENDING_MODE = "Select Screen Blending Mode" # Select Screen Blending Mode
+    SELECT_COLOR_DODGE_BLENDING_MODE = "Select Color Dodge Blending Mode" # Select Colour Dodge Blending Mode
+    SELECT_LINEAR_DODGE_BLENDING_MODE = "Select Linear Dodge Blending Mode" # Select Linear Dodge Blending Mode
+    SELECT_OVERLAY_BLENDING_MODE = "Select Overlay Blending Mode" # Select Overlay Blending Mode
+    SELECT_HARD_OVERLAY_BLENDING_MODE = "Select Hard Overlay Blending Mode" # Select Hard Overlay Blending Mode
+    SELECT_SOFT_LIGHT_BLENDING_MODE = "Select Soft Light Blending Mode" # Select Soft Light Blending Mode
+    SELECT_HARD_LIGHT_BLENDING_MODE = "Select Hard Light Blending Mode" # Select Hard Light Blending Mode
+    SELECT_VIVID_LIGHT_BLENDING_MODE = "Select Vivid Light Blending Mode" # Select Vivid Light Blending Mode
+    SELECT_LINEAR_LIGHT_BLENDING_MODE = "Select Linear Light Blending Mode" # Select Linear Light Blending Mode
+    SELECT_PIN_LIGHT_BLENDING_MODE = "Select Pin Light Blending Mode" # Select Pin Light Blending Mode
+    SELECT_HARD_MIX_BLENDING_MODE = "Select Hard Mix Blending Mode" # Select Hard Mix Blending Mode
+    SELECT_DIFFERENCE_BLENDING_MODE = "Select Difference Blending Mode" # Select Difference Blending Mode
+    SELECT_EXCLUSION_BLENDING_MODE = "Select Exclusion Blending Mode" # Select Exclusion Blending Mode
+    SELECT_HUE_BLENDING_MODE = "Select Hue Blending Mode" # Select Hue Blending Mode
+    SELECT_SATURATION_BLENDING_MODE = "Select Saturation Blending Mode" # Select Saturation Blending Mode
+    SELECT_COLOR_BLENDING_MODE = "Select Color Blending Mode" # Select Colour Blending Mode
+    SELECT_LUMINOSITY_BLENDING_MODE = "Select Luminosity Blending Mode" # Select Luminosity Blending Mode
```

Comment:
To jest do wyrzucenia, bo jest już obsługiwane przez odpowiedni kontroler (i jest to szersze, bo obsługiwane są wszystkie tryby w kricie)